src/routes/RepositoriesList/RepositoriesList.tsx

```diff
@@ -164,32 +196,29 @@ export default function RepositoriesList() {
     // clearing previous states
     setRepositoryList([]);
     setSelectedRepoNames([]);
-
-    const listOfOrgNames = [];
     if (userOrgs) {
       // check if view is global vs scoped to a organization
-      if (currentOrg === null) {
-        userOrgs.map((org) => listOfOrgNames.push(org.name));
-        // add user to fetch user specific repositories
-        listOfOrgNames.push(currentUser.username);
-      } else {
-        listOfOrgNames.push(currentOrg);
+      const listOfOrgNames: string[] = props.currentOrg
+        ? [props.currentOrg]
+        : userOrgs.map((org) => org.name);
+
+      try {
+        const repos = await fetchAllRepos(listOfOrgNames);
+        // TODO: Here we're formatting repo's into the correct
+        // type. Once we know the return type from the repo's
+        // API we should pass 'repos' directly into 'setRepositoryList'
+        const formattedRepos = repos.map((repo) => ({
+          namespace: repo.namespace,
+          name: repo.name,
+          is_public: repo.is_public,
+          last_modified: repo.last_modified,
+        }));
+        setRepositoryList(formattedRepos);
+      } catch (err) {
+        console.log('setting err');
```

Comment:
Can you remove any of the `console.log` messages?

Reply:
oops, this one snuck by. Removed.